admin/console-rails-shell.include.rst

```diff
@@ -5,3 +5,6 @@
 
    # source installation
    $ rails c
+
+   # docker & docker-compose installation
```

Comment:
Please orientate on our other console command lines. They're all starting with `$ `.
I understand that this may be irritating but is this way in the whole documentation.

Also, I'd suggest to replace the variable with a place holder like e.g. `{placeholder}`.
The variable approach may confuse our some users  as they might assume it's being set automatically.